vault.py

```diff
@@ -57,12 +57,17 @@ def run(self, terms, inject=None, variables=None, **kwargs):
         if not url:
             raise AnsibleError('VAULT_ADDR environment variable is missing')
 
-        token = os.getenv('VAULT_TOKEN')
+        # the environment variable overrides file-based token
+        token = os.getenv('VAULT_TOKEN') or open(os.getenv("HOME") + "/.vault-token").read()
```

Comment:
Sorry for the delay. I'm unlikely to get further on this, as I'm not a python dev. It looks like @richfromm's patch is more complete anyway.